src/controllers/api/sellController.ts

```diff
@@ -45,6 +45,11 @@ export const sellController: RequestHandler = async (req, res) => {
             inventory.Melee.pull({ _id: sellItem.String });
         });
     }
+    if (payload.Items.SpecialItems) {
+        payload.Items.SpecialItems.forEach(sellItem => {
+            inventory.SpecialItems.pull({ _id: sellItem.String });
+        });
+    }
```

Comment:
As far as I can tell, it is not possible to sell exalted weapons, so this doesn't seem needed.

Reply:
Exalted Weapons don't appear in your inventory on live, so it *should* be impossible to sell them without selling the Warframe itself.

Reply:
I'm having my doubts that even selling the warframe would sell the exalted weapon as multiples of the same warframe use the same exalted weapon.

Reply:
I added this to be able to get rid of duplicates via web

Reply:
Getting duplicates was a bug which has now been fixed (#645).